lib/ui/window/platform_message_response_dart.cc

```diff
@@ -7,12 +7,41 @@
 #include <utility>
 
 #include "flutter/common/threads.h"
+#include "flutter/lib/ui/window/window.h"
 #include "lib/fxl/functional/make_copyable.h"
 #include "lib/tonic/dart_state.h"
 #include "lib/tonic/logging/dart_invoke.h"
 
 namespace blink {
 
+namespace {
+
+// Avoid copying the contents of messages beyond a certain size.
+const int kMessageCopyThreshold = 100;
```

Comment:
How about making this one page sized instead?

Reply:
This was based on measurement of the tradeoff between allocating (and later deleting) the new vector versus doing the memcpy.

@rmacnak-google do external objects have any additional performance cost compared to a standard TypedData?

Reply:
Yes, the size of the weak handle (~4 words) and the time for the GC to look at the weak handle table to see if they have been collected (a few dozen instructions). I'd expect it to pay off before one page, but even a high threshold here I think is fine because the bulk of the benefit comes from the very large messages.

Reply:
Increased the threshold to 1000 bytes